dev/ProgressRing/ProgressRing.cpp

```diff
@@ -90,10 +230,10 @@ void ProgressRing::UpdateSegment()
 
 void ProgressRing::UpdateRing()
 {
-    const double thickness = StrokeThickness();
-    const auto size = ComputeEllipseSize(thickness, ActualWidth(), ActualHeight());
+    const double thickness = GetStrokeThickness();
+    const auto size = GetCircleRadius(thickness, ActualWidth());
 
-    const float segmentWidth = size.Width;
+    const float segmentWidth = size;
```

Comment:
SegmentWidth and Size are the same value, why do you need both?

Reply:
Also, both names seem wrong, the value is the radius, not the size or the width

---
In reply to: [381003846](https://github.com/microsoft/microsoft-ui-xaml/pull/1858#discussion_r381003846) [](ancestors = 381003846)